docs/conversion_examples_gallery/behavior/movie.rst

```diff
@@ -0,0 +1,37 @@
+Video data conversion (multimedia formats)
+------------------------------------------
+
+Install NeuroConv with the additional dependencies necessary for reading multimedia data.
+
+.. code-block:: bash
+
+    pip install neuroconv[movie]
+
+This interface can handle conversions from avi, mov, mp4, wmv, flv and most FFmpeg_ supported formats to NWB using the
+:py:class:`~neuroconv.datainterfaces.behavior.movie.moviedatainterface.MovieInterface` class.
+
+.. _FFmpeg: https://ffmpeg.org/
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from tabnanny import verbose
```

Comment:
what is the purpose of this import?

Reply:
This is wrong. Thanks for spotting that.

Reply:
Removed.